docs/fast_data/configuration/single_view_creator.md

```diff
@@ -403,8 +403,10 @@ This feature is supported from version `5.6.1` of the Single View Creator
 To configure a Single View Creator dedicated to [Single View Patch](/fast_data/configuration/single_views.md#single-view-patch) operations, some steps have to be followed:
 
 * Set the env var `KAFKA_PROJECTION_UPDATE_TOPICS` with the comma separated list of the `pr-update` topics corresponding to the SV-Patch Projection.
+* Set the env var `PROJECTIONS_CHANGES_SOURCE` to `KAFKA` so the consumer is summoned.
+* Configure the environment variables for kafka (see the [environment variables table](#environment-variables))
```

Comment:
which are all the KAFKA env vars? are they stated somewhere?

Reply:
You're right I added a paragraph explaining what is needed

Reply:
is it done?